R/models.R

```diff
@@ -10,17 +10,22 @@
 #' @export
 #' @import data.table
 #' @importFrom utils read.csv
+#' @examples 
+#' par=Parameters$new()
+#' d = loadData(par, includeVaccines = 'BCG')
+#' d = loadData(par,candidatefile = 
+#'              "inst/extdata/vaccinesSummary.csv")
 loadData <- function(par, candidateFile=NULL, includeVaccines=c()) {
   # We set encoding to BOM UTF to avoid cross-platform issues
 
   if (is.null(candidateFile)) {
     if (par$inputfile=="Default") {
-      d <- data.table(read.csv("Data/vaccinesSummary.csv", fileEncoding = "UTF-8-BOM"))
+      d <- data.table(read.csv(system.file("extdata/vaccinesSummary.csv", package = 'vaccineEarlyInvest'), fileEncoding = "UTF-8-BOM"))
```

Comment:
I think you can't use `/` in `system.file()`, please check documentation. It would be `system.file("extdata", "vaccinesSummary.csv")`

Reply:
I've tested this syntax before uploading and it's an alternative of the `system.file("extdata", "vaccinesSummary.csv"). The reason for using this is to make function input simpler.